examples/plot_dem_hillshade.py

```diff
@@ -0,0 +1,139 @@
+"""
+Create A Hillshade Layer Using EarthPy
+========================================
+
+Learn how to create a hillshade Layer from a DEM using EarthPy
+"""
+
+###############################################################################
+# Create A Hillshade Layer Using EarthPy
+# ---------------------------------------
+#
+# .. note::
+#       The examples below will show you how to use the ``es.hillshade()``
+#       function from EarthPy.
+
+###############################################################################
+# Create hillshade from DEM
+# -----------------------------
```

Comment:
A hillshade is a 3D representation of a surface. Hillshades are generally rendered in greyscale. The darker and lighter colors represent the shadows and highlights that you would visually expect to see in a terrain model. Hillshades are often used as an underlay in a map, to make the data appear more 3-Dimensional and thus visually interesting. This vignette will show you how to create a hillshade from a DEM using EarthPy.  It will highlight how to adjust the azimuth and other settings that will impact how the shadows are modeled in the data. 

The hillshade function is a part of the spatial module in EarthPy. To begin using the EarthPy ``hillshade()`` function, import the needed ...

Reply:
@lwasser thanks for writing this up, I was having a really hard time with what to say in this section, and this is so much better!